packages/bsky/src/indexer/services.ts

```diff
@@ -0,0 +1,25 @@
+import { IdResolver } from '@atproto/identity'
+import Database from '../db'
+import { Labeler } from '../labeler'
+import { BackgroundQueue } from '../background'
+import { IndexingService } from '../services/indexing'
+import { LabelService } from '../services/label'
+
+export function createServices(resources: {
+  idResolver: IdResolver
+  labeler: Labeler
+  backgroundQueue: BackgroundQueue
+}): Services {
+  const { idResolver, labeler, backgroundQueue } = resources
+  return {
+    indexing: IndexingService.creator(idResolver, labeler, backgroundQueue),
+    label: LabelService.creator(),
+  }
+}
+
+export type Services = {
```

Comment:
may be cool to have the actual services live in this dir as well

def not a blocking thing tho

Reply:
That would def be ideal.  I think there are some cross-dependencies with the label service, which is also used in the api server.  If we can resolve those then I think we're good to move some services around.  I was thinking of this as an "all or nothing" kind of thing just to avoid confusion.

Reply:
yeah i think it's totally fine the way it is

maybe we could split the labeling service into a creation & reading